packages/suite/src/components/suite/modals/ReviewTransaction/components/getIsUpdatedSendFlow.ts

```diff
@@ -0,0 +1,21 @@
+import { TrezorDevice } from '@suite-types/index';
+import { DeviceModel, getDeviceModel } from '@trezor/device-utils';
+import { versionUtils } from '@trezor/utils';
+
+export const getIsUpdatedSendFlow = (device: TrezorDevice) => {
+    const { firmwareRelease } = device;
+    if (!firmwareRelease) {
+        return false;
+    }
+
+    const model = getDeviceModel(device);
+
+    const {
+        release: { version },
+    } = firmwareRelease;
```

Comment:
`firmwareRelease` is about available FW releases for update, it's not saying what is the current FW version on the device. For that you have to check `features`

Reply:
use [`getFirmwareVersion`](https://github.com/trezor/trezor-suite/blob/feat/new-send-flow/packages/device-utils/src/firmwareUtils.ts/#L6)

Reply:
I guess it can work like this:
```typescript
import { TrezorDevice } from '@suite-types/index';
import { getFirmwareVersion } from '@trezor/device-utils';
import { versionUtils } from '@trezor/utils';

export const getIsUpdatedSendFlow = (device: TrezorDevice) => {
    const firmwareVersion = getFirmwareVersion(device);

    return versionUtils.isNewerOrEqual(firmwareVersion, '2.6.0');
};

```

but I would expect to have it defined on a different level - somewhere in Connect as a feature, I guess? What do you think @mroz22, is this acceptable?

Reply:
7b165ac6d

Reply:
check for device model is now redundant

Reply:
2cbf0e148

Reply:
ok, let's live with that until we reconsider the ButtonRequest update to make more reliable